flocker/node/agents/test/test_blockdevice.py

```diff
@@ -865,6 +1026,76 @@ class DiscoverVolumesMethod(Names):
     INFER_VOLUMES_FROM_STATE = NamedConstant()
 
 
+@implementer(ILocalState)
+class FakeBlockDeviceDeployerLocalState(PClass):
```

Comment:
This is definitely not ideal. But the tests below were copied from the real tests and just adjusted enough to construct this. I wanted to avoid making this PR bigger by adjusting the `calculate_changes` tests as well. That will be done in a follow up.